pom.xml

```diff
@@ -407,6 +402,17 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <!-- Version specified in parent POM -->
+        <configuration>
+          <systemPropertyVariables>
+            <java.util.logging.config.class>org.jenkins.tools.test.logging.LoggingConfiguration</java.util.logging.config.class>
+          </systemPropertyVariables>
+          <runOrder>alphabetical</runOrder>
```

Comment:
Unrelated code cleanup: getting the tests to be run in a consistent order. The plugin parent POM has this, but the `jenkinsci/pom` does not (though Jenkins core specifies it). It would be nice to move this up from here and from core into `jenkinsci/pom`, but I am explicitly not taking on this task as part of the present PR.